src/manifests_workflow/input_manifests.py

```diff
@@ -212,6 +217,24 @@ def add_to_cron(self, version: str) -> None:
 
         logging.info(f"Wrote {jenkinsfile}")
 
+    def add_to_integTest_notification_cron(self, version: str) -> None:
+        logging.info(f"Adding new version to integ test notification cron: {version}")
+        integTestJenkinsfile = self.cron_integTest_notification_jenkinsfile()
+        with open(integTestJenkinsfile, "r") as f:
+            data = f.read()
+
+        cron_entry = f"H */6 * * * %INPUT_MANIFEST={version}/{self.prefix}-{version}.yml\n"
+
+        if cron_entry in data:
+            raise ValueError(f"{integTestJenkinsfile} already contains an entry for {self.prefix} {version}")
+
+        data = data.replace("parameterizedCron('''\n", f"parameterizedCron('''\n{' ' * 12}{cron_entry}")
```

Comment:
Parenthesis wrong?

See above example
```
        data = data.replace("parameterizedCron '''\n", f"parameterizedCron '''\n{' ' * 12}{cron_entry}")

```

Reply:
Its different for both files. Both are correct. Let me know if you have a preference